packages/gluestick-cli/index.js

```diff
@@ -47,6 +47,31 @@ commander
   });
 
 commander
+  .command('watch')
+  .description('applying changes in gluestick folder')
+  .action(() => {
+    let srcPath = require(path.join(process.cwd(), 'package.json')).dependencies.gluestick;
```

Comment:
This may fail if CWD is not valid project, so this should be wrapped into try/catch block and add more meaningful message like:
- _Cannot find package.json in `<process.cwd()>`_ if require fails
- _Path is not a valid GlueStick project_ if `gluestick` is not a dependency

Also let's check if this depenedency is a path, not version string with some regex.